server/index.js

```diff
@@ -74,7 +74,7 @@ app.post('/api/auth/sign-in', (req, res, next) => {
     .catch((err) => next(err));
 });
 
-app.post('/api/projects', (req, res, next) => {
+app.post('/api/project/newproject', (req, res, next) => {
```

Comment:
I'm curious why you changed the name of this endpoint? It's pretty standard in REST servers to use the same path in the requests but differentiate the action based on the method type. So I would actually expect `POST /api/projects` to create a new project.